### PR TITLE
update score of interop '23 accessibility investigation

### PR DESCRIPTION
**Pull request by cookiecrook**
## Description
Percentage update (87% -> 91%) for Accessibility Interop Investigation, based on scoring criteria:
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/3

### CODE REVIEW

**Comment by cookiecrook** [comment]
@boazsender

File: webapp/components/interop-data.js [hunk 1]
```diff
@@ -320,7 +320,8 @@ export const interopData = {
           { 'date': '2023-05-13', 'score': 700 },
           { 'date': '2023-06-27', 'score': 780 },
           { 'date': '2023-09-05', 'score': 860 },
-          { 'date': '2023-09-27', 'score': 870 }
+          { 'date': '2023-09-27', 'score': 870 },
+          { 'date': '2023-10-12', 'score': 920 }
```
Comment (cookiecrook) [inline webapp/components/interop-data.js:324]:
```suggestion
          { 'date': '2023-10-12', 'score': 910 }
```

File: webapp/static/interop-data.json [hunk 2]
```diff
@@ -301,7 +301,8 @@
           { "date": "2023-05-13", "score": 700 },
           { "date": "2023-06-27", "score": 780 },
           { "date": "2023-09-05", "score": 860 },
-          { "date": "2023-09-27", "score": 870 }
+          { "date": "2023-09-27", "score": 870 },
+          { "date": "2023-10-12", "score": 920 }
```
Comment (cookiecrook) [inline webapp/static/interop-data.json:305]:
```suggestion
          { "date": "2023-10-12", "score": 910 }
```

**Review by cookiecrook — COMMENTED** [review]
Miscounted the first time. Corrected!

**Review by boazsender — APPROVED** [review]

**Comment by cookiecrook** [comment]
@boazsender @KyleJu 
```
Error: Error thrown outside of test function: Blocked a frame with origin "http://localhost:8081" from accessing a cross-origin frame.
  ChildRunner.loaded at /components/wct-browser-legacy/browser.js:633
           <unknown> at /components/wct-browser-legacy/browser.js:611
```

How to pass the unrelated(?) check failure?

File: webapp/static/interop-data.json [hunk 3]
```diff
@@ -301,7 +301,8 @@
           { "date": "2023-05-13", "score": 700 },
           { "date": "2023-06-27", "score": 780 },
           { "date": "2023-09-05", "score": 860 },
-          { "date": "2023-09-27", "score": 870 }
+          { "date": "2023-09-27", "score": 870 },
+          { "date": "2023-10-12", "score": 910 }
```
Comment (cookiecrook) [inline webapp/static/interop-data.json:305]: Another 1% merged in this morning.

```suggestion
          { "date": "2023-09-27", "score": 870 },
          { "date": "2023-10-12", "score": 910 },
          { "date": "2023-10-13", "score": 920 }
```

File: webapp/components/interop-data.js [hunk 4]
```diff
@@ -320,7 +320,8 @@ export const interopData = {
           { 'date': '2023-05-13', 'score': 700 },
           { 'date': '2023-06-27', 'score': 780 },
           { 'date': '2023-09-05', 'score': 860 },
-          { 'date': '2023-09-27', 'score': 870 }
+          { 'date': '2023-09-27', 'score': 870 },
+          { 'date': '2023-10-12', 'score': 910 }
```
Comment (cookiecrook) [inline webapp/components/interop-data.js:324]: Another 1% merged in this morning.

```suggestion
          { 'date': '2023-09-27', 'score': 870 },
          { 'date': '2023-10-12', 'score': 910 },
          { 'date': '2023-10-13', 'score': 920 }
```

**Comment by cookiecrook** [comment]
Sorry, I committed an update (+1% this morning) which dropped the prior check failure details link. I should have done this first and then asked the question if it failed again.

**Comment by cookiecrook** [comment]
Seems resolved. Please disregard.

**Comment by cookiecrook** [comment]
We might get another +1% by EOD, so I'll recycle the review then.

**Comment by cookiecrook** [comment]
[That one](https://github.com/web-platform-tests/wpt/pull/42412) is not going to happen today, so recycling review + automerge.

**Review by boazsender — APPROVED** [review]

**Comment by cookiecrook** [comment]
@boazsender any idea why merging is blocked? You are the approving reviewer, but review is still required?

**Comment by KyleJu** [comment]
> @boazsender any idea why merging is blocked? You are the approving reviewer, but review is still required?

I stamped it and it should work now. Unsure why it was blocked

**Comment by cookiecrook** [comment]
Just FYI I don't have merge access so one of you will need to do it. Thanks @KyleJu and @boazsender

**Comment by KyleJu** [comment]
> Just FYI I don't have merge access so one of you will need to do it. Thanks @KyleJu and @boazsender

Done!

**Pull request merged**